### DISCUSSION TITLE
How to download a list of users? YAML file difficulty

### DISCUSSION THREAD

**Discussion by BluesFanUK** [Serene-Arc/bulk-downloader-for-reddit#897] [General]
The instructions seem to be a little more geared towards experienced users. I just went by the example and created a YAML file with the same name (my_opts.yaml), and used the same line of code to run it, but all I get is: **[2023-06-24 17:37:38,968 - root - INFO] - Program complete**

Clearly i'm being stupid, but I presumed feeding a YAML file into CMD with the list of users would just go through the users and download for me. Can this not be done with a text file? The instructions example has the the following:
_cat users.txt | xargs -L 1 echo --user | xargs -L 50 bdfr download <ARGS>_

I'vr tried that, but xargs isn't mentioned as part pf the initial setup, so this doesn't work at all.

Help please.

EDIT - So replacing subreddit with user seems to break it. It doesn't work with users for some reason, works fine with subreddits.

**Comment by Serene-Arc** [comment]
There's no YAML file in your command? Additionally you at least need to specify a download location. It's hard to tell what's going on past that if you don't share the logs.

**Reply by BluesFanUK** [reply]
It looks like you can't download a list of redditors, it only works with a subreddit, here's my steps (never heard of a YAML file till now.

1. https://codebeautify.org/yaml-editor-online
2. Copy the contents of your example to the above link, add my own changes (see below)
3. Download as YAML (named as optionsfile.yaml)

In the folder where this file resides i've typed the following into CMD:  **bdfr download ./Reddit/ --opts optionsfile.yaml**

So this should run the downloader, and save it to a folder in /Reddit.

This is returned:
```
G:\>bdfr download ./Reddit/ --opts optionsfile.yaml
Ignoring an unknown YAML argument: redditor
[2023-07-08 15:29:13,810 - root - INFO] - Program complete
```
My YAML File looks like this - it seems to work ok for a subreddit but not a redditor.

```

file_scheme: "{DATE}_{REDDITOR}_{TITLE}_{POSTID}"
limit: 1000
no_dupes: true
redditor:
  - pennycakez
```

If I try changing redditor to user, i get this:

```
G:\>bdfr download ./Reddit/ --opts optionsfile.yaml
[2023-07-08 15:31:51,173 - root - INFO] - Program complete
```

If we have to use the other command line to get users (as below), what is xargs, how do we get it and what goes in the text file for Windows users?
**cat users.txt | xargs -L 1 echo --user | xargs -L 50 bdfr download**
